modules/distr-tracing-deploy-otel-collector.adoc

```diff
@@ -0,0 +1,127 @@
+////
+This module included in the following assemblies:
+- distr_tracing_install/distr-tracing-deploying.adoc
+////
+
+[id="distr-tracing-deploy-otel-collector_{context}"]
+= Deploying distributed tracing data collection
+
+The custom resource definition (CRD) defines the configuration used when you deploy an instance of {OTELName}.
```

Comment:
Not critical, but would it help to specify which CRD defines the config for otel?